templates/pimcore5/files/.platform.app.yaml

```diff
@@ -0,0 +1,100 @@
+# This file describes an application. You can have multiple applications
+# in the same project.
+#
+# See https://docs.platform.sh/user_guide/reference/platform-app-yaml.html
+
+# The name of this app. Must be unique within a project.
+
+
+# The runtime the application uses.
+name: pimcore
+type: php:7.3
+
+
+# The relationships of the application with services or other applications.
+#
+# The left-hand side is the name of the relationship as it will be exposed
+# to the application in the PLATFORM_RELATIONSHIPS variable. The right-hand
+# side is in the form `<service name>:<endpoint name>`.
+relationships:
+  database: "pimcore_mysqldb:pimcore_admin"   
+  redis: 'rediscache:redis'
+
+# Add additional PHP extensions.
+build:
+  flavor: none
+
+runtime:
+  extensions:
+    - redis
+    - iconv
+    - dom
+    - simplexml
+    - exif
+    - opcache
+    - imagick
+    - fileinfo
+
+variables:
+  env:
+    PIMCORE_INSTALL_ADMIN_USERNAME: "admin"
+    PIMCORE_INSTALL_ADMIN_PASSWORD: "admin"
```

Comment:
These worry me considerably.  Does this have any impact after the initial install?  If not, let's include a comment that users should remove those after installation.  If it does, then we can't have them here at all because it's a security hole.

Reply:
It has no impact after initial install, I'll add a comment explaining that it should be removed after the setup.

Reply:
I'd check with the security team as well and get their signoff.  I don't want to ship this without their OK.

Reply:
So, thinking about this. I can take example on the wagtail template and explain in the template README.md that the user has to log in SSH and execute the install script where he has to set the admin user_name and the admin_password. Anyway, if it's not done, accessing the site will show you an explicit message saying that pimcore is not installed.

Reply:
As long as the behavior is as @RibanKeke describes and is documented as described, I think that is an acceptable approach to this.